src/libYARP_manager/src/localbroker.cpp

```diff
@@ -885,8 +889,14 @@ int LocalBroker::ExecuteCmd(void)
         //int saved_stderr = dup(STDERR_FILENO);
         dup2(pipe_to_stdout[WRITE_TO_PIPE], STDOUT_FILENO);
         dup2(pipe_to_stdout[WRITE_TO_PIPE], STDERR_FILENO);
-        fcntl(STDOUT_FILENO, F_SETFL, fcntl(STDOUT_FILENO, F_GETFL) | O_NONBLOCK);
-        fcntl(STDERR_FILENO, F_SETFL, fcntl(STDERR_FILENO, F_GETFL) | O_NONBLOCK);
+        if (fcntl(STDOUT_FILENO, F_SETFL, fcntl(STDOUT_FILENO, F_GETFL) | O_NONBLOCK) == -1) {
```

Comment:
Instead of `return 0`, in case of error I would return a different number, 0 usually means "everything is ok"

Reply:
Actually, this function is returning 0 everywhere, so I suppose it's fine here...
@Nicogene you can eventually change the function later, I'm going to merge this now!